python/tk_multi_workfiles/file_model.py

```diff
@@ -377,7 +377,7 @@ def get_cached_file_versions(self, key, work_area, clean_only=False):
                             searching for files.
         :returns:           A dictionary {version:FileItem} of all file versions found.
         """
-        return self._search_cache.find_file_versions(work_area, key)
+        return self._search_cache.find_file_versions(work_area, key, include_dirty=not clean_only)
```

Comment:
it looks a little bit funny that this method has a `clean_only` param only to call another method which has a `include_dirty` parameter. All that it's doing is to negate the flag and call the other method. Makes me wonder if these methods could be combined into a single one? I don't have the proper big picture so may not make sense at all, but i thought it's worth at least checking!